smallrye-reactive-messaging-amqp/src/main/java/io/smallrye/reactive/messaging/amqp/AmqpConnector.java

```diff
@@ -295,6 +310,16 @@ void init() {
                 .ignore();
     }
 
+    public List<String> getClientCapabilities(AmqpConnectorCommonConfiguration configuration) {
+        if (configuration.getCapabilities().isPresent()) {
+            String capabilities = configuration.getCapabilities().get();
+            return Arrays.stream(capabilities.split(","))
+                    .filter(s -> !s.isEmpty())
+                    .map(String::trim).collect(Collectors.toList());
```

Comment:
you should invert the two statements as you would not filter out " "

Reply:
That's right!